src/EditorFeatures/Test/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -199,138 +330,6 @@ protected Task TestSmartTagGlyphTagsAsync(string initialMarkup, ImmutableArray<s
             }
         }
 
-        protected async Task TestAddDocumentInRegularAndScriptAsync(
```

Comment:
Code deleted here moved into a separate partial declaration that is not linked into CodeStyle layer.